contents/blog/the-posthog-array-1-28-0.md

```diff
@@ -0,0 +1,125 @@
+---
+date: 2021-09-15
+title: Array 1.28.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: release-notes
+featuredImage: ../images/blog/array/1-28-0.png
+excerpt: PostHog 1.28.0 has launched! Significantly revamped performance for slower queries, advanced engagement cohorts, SAML support, advanced funnels and many more improvements and fixes.
+---
+
+PostHog 1.28.0 has launched! 
+
+Significantly revamped performance for slower queries, advanced engagement cohorts, SAML support, advanced funnels and many more improvements and fixes.
+
+### Community MVP 🏆
+
+Thanks to all our community members for helping move PostHog forward! This release cycle's Community MVP goes to [manish001in](https://github.com/manish001in)!
+
+Manish helped us pushed a PR that ensures tab titles are properly set when navigating between dashboards (PR [#5822](https://github.com/PostHog/posthog/pull/5822)) and is also pushing a PR to make it easy to copy properties from the person page (PR [#5884](https://github.com/PostHog/posthog/pull/5884)).
+
+### Help us improve PostHog
+
+We’re working hard to improve the PostHog experience and would love to talk to you! Please join one of our Product, Engineering or Marketing team members on a quick 30-min call to help us understand how to improve. Schedule directly [on Calendly](https://calendly.com/posthog-feedback).
+
+As a small thank you for your time, we're giving away some awesome [PostHog merch](https://merch.posthog.com)!
+
+## PostHog 1.28.0 Release Notes
```

Comment:
Release notes should be the first thing as that's what most people will care about

Reply:
done!